data/human/human missions.txt

```diff
@@ -3996,3 +3996,91 @@ mission "Fumbling Mugger"
 event "fumbling mugger followup timer"
 
 
+
+mission "Dangerous Games 1"
+	name "Dragonriders"
+	description "A lively man named Karengo and a few passionate youth are traveling with you to see the dragons on <stopover>. Take the group to <destination> and return the group back to <source> once they've finished."
+	minor
+	passengers 9
+	to offer
+		random < 15
+	source 
+		attributes frontier pirate rim
+	stopover Skymoot
+	on offer
+		conversation
+			`	Upon entering the spaceport, you notice that the local bar seems to be more active than usual. The moment you pass through the bar door, a strong drink is pressed into your hand. You look at it, dumbfounded, and then look up to the waiter who gave it to you. "Compliments of Mr. Karengo," he says, nodding his head towards a crowd of people, mostly young men, before moving on.`
+			choice
+				`	(Head over to "Mr. Karengo").`
+				`	(Leave).`
+					decline
+			`	As you make your way over, you note that nearly everyone in the bar has the same drink you do. In the middle of the crowd, a large, heavily muscled, and tattooed man is gesticulating wildly, apparently telling a story.`
+			choice
+				`	(Go listen to the man).`
+				`	(Leave).`
+					decline
+			`	"...so then he asks me, have you ever ridden one? And I look back at this crazy joithead and I say, you're a crazy joithead. But he's got me thinking, y'know? Nobody has ever ridden one. Least, not nobody that could talk after." He lets out a sharp laugh, then raises his voice. "So I ask you all; who wants to join Karengo to make history? Who wants to ride a dragon?"`
+			`	You've heard the stories of the dragons flying through <planet>'s skies as a child. Great beasts; graceful and dangerous, the apex predators of the air; they are strikingly similar to the dragons described in ancient legends. <planet> is host to many dangerous creatures, and these living myths hold a place at the top of that list.`
```

Comment:
```suggestion
			`	A cheer goes up from the men, especially the drunk ones. Karengo starts to hand out death and accident waivers and collect money for the expedition. Suddenly, his expression darkens as he cocks his head, listening to an earpiece. "Listen up!" he yells, "Seems my regular pilot got in the way of some less reputable folk, and my replacement pilot misplaced his balls this evening and can't find 'em! We need ourselves a ship and a captain to bring this lot to <stopovers>! Who's brave enough? Who else wants to make history?"`
```


This'll become: `Skymoot in the Hadar system`, assuming I remember right and Hadar is the correct system, I'm not entirely sure. It might not appear right that he'd say it like this, so may e just `Skymoot` is better.

Reply:
It appears that you've made two comments on one line. Which one should I commit?

Reply:
Ignore this one.